lms/templates/ux/reference/legacy/course-skeleton.html

```diff
@@ -0,0 +1,945 @@
+## mako
```

Comment:
There was no v1-style course skeleton, so I decided it was worth adding one to make sure that the alerts render correctly there too.